static/src/plugins/__tests__/toaster.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { createApp } from "vue";
+import Toaster from "../toaster";
+
+describe("Toast plugin", () => {
+  it("works as plugin", () => {
+    const app = createApp({});
+
+    app.use(Toaster);
+    app.mount("body");
+    app.config.globalProperties.$toast.success("test message", {
+      duration: 1,
+    });
+
+    expect(document.body.querySelectorAll("div > div#container").length).equal(
+      1
+    );
```

Comment:
```suggestion
    expect(document.body.querySelectorAll("div > div#container")).not.to.be.empty;
```
Maybe? Idk

Reply:
Yeah the equal formats ugly, so would be nice to have another way but your suggestion would also accept 2 or more matches. I think it would be nice if the test also catches that there are not too many toasts created on a single call